src/main/kotlin/graphics/scenery/primitives/Atmosphere.kt

```diff
@@ -108,18 +123,20 @@ open class Atmosphere(initSunDir: Vector3f? = null, emissionStrength: Float = 0.
         if (mapping != null) {
             val (angle, axis) = mapping
             val rotation = Quaternionf().rotationAxis(toRadians(angle.toDouble()).toFloat(), axis.x, axis.y, axis.z)
-            sunDir.rotate(rotation)
+            sunDirection.rotate(rotation)
         }
     }
 
     /** Attach Up, Down, Left, Right key mappings to the inputhandler to rotate the sun in increments.
      * Keybinds are Ctrl + cursor keys for fast movement and Ctrl + Shift + cursor keys for slow movement.
      * @param increment Increment value for the rotation in degrees, defaults to 20°. Slow movement is always 10% of [increment]. */
-    fun attachRotateBehaviours(inputHandler: InputHandler, increment: Float = 20f) {
+    fun attachRotateBehaviors(inputHandler: InputHandler, increment: Float = 20f) {
```

Comment:
More nitpicking: I'd just call it `attach/detachBehaviours`, as there are no others.